client/src/procedures/primitives.rs

```diff
@@ -347,17 +350,56 @@ impl FromStr for MnemonicLanguage {
     }
 }
 
+pub mod serde_bip39_mnemonic {
+    use super::bip39;
+
+    pub fn serialize<S>(m: &bip39::Mnemonic, s: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        s.serialize_str(m.as_ref())
```

Comment:
I know exactly how I feel about it. There's no workaround.

Reply:
Maybe serde is not even needed, I removed it locally and it's fine

Reply:
What exactly did you remove?

Reply:
What exactly did you remove?

Reply:
Let's ignore for now

Reply:
I think we can keep serde stuff just in case. If it's not used then there's no issue. If it's used by an external crate then at least we don't break compatibility for them.